apigenerator/resources/1 - Project/1 - BaseProject/Project/src/e_Infra/h_MCP/LlmServiceFactory.py

```
@@ -93,9 +89,9 @@ def get_llm_service(x_provider_header: str = None) -> LlmServiceBase:
 
         if temperature is None: # Fallback to environment default temperature if runtime was missing or invalid
             env_temp_str = None
-            if chosen_provider_name == "gemini": env_temp_str = ENV_DEFAULT_GEMINI_TEMPERATURE
-            elif chosen_provider_name == "openai": env_temp_str = ENV_DEFAULT_OPENAI_TEMPERATURE
-            elif chosen_provider_name == "anthropic": env_temp_str = ENV_DEFAULT_ANTHROPIC_TEMPERATURE
+            if chosen_provider_name == "gemini": env_temp_str = os.environ.get('ENV_DEFAULT_GEMINI_TEMPERATURE')
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)